en/config/actuators.md

```diff
@@ -1,65 +1,170 @@
 
-:::note
-**Positions are relative to the Center of Gravity of the vehicle, and NOT the Autopilot location!**
-:::
+#### Motor Geometry: VTOL Tailsitter
+
+The default tailsitter motor geometry is shown below.
+Motors have the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+
+![Geometry motor: tailsitter vtol](../../assets/config/actuators/qgc_geometry_tailsitter_motors.png)
+
+#### Motor Geometry: VTOL Tiltrotor
+
+The default VTOL tiltrotor motor geometry is shown below.
+
+![Geometry motor: tiltrotor vtol](../../assets/config/actuators/qgc_geometry_tiltrotor_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate which servo is used to tilt the motor:
+
+- `Tilted by`: The associated servo used for tilting the motor.
+  The properties of this servo are defined in the [Motor Tilt Servo Geometry](#motor-tilt-servo-geometry).
+
+
+#### Motor Geometry: Standard VTOL
+
+The default standard VTOL ("quadplane") motor geometry is shown below.
+
+![Geometry motor: standard vtol](../../assets/config/actuators/qgc_geometry_standard_vtol_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate the direction in which the motor moves the vehicle (for a standard VTOL, the hover motors are usually set "upwards" and the pusher motor is set to "forwards").
+
+- `Axis`: One of `Upwards`, `Downwards`, `Forwards`, `Backwards`, `Leftwards`, `Rightwards`, `Custom`
+  - If `Custom` is selected, then the UI displays three additional fields for setting the motor orientation.
+
+#### Motor Geometry: Other Vehicles
+
+Other vehicle types will define an appropriate motor geometry for their frame type.
+Once again these motors will generally have the same kinds of properties as shown above.
+
+For example, a fixed-wing vehicle may just have a single pusher moter, while a rover with differential steering will have a motor for throttle and for steering.
+
+#### Motor Position Coordinate System
 
-![Actuators CG reference diagram](/assets/config/actuators/Quadcopter_Actuators_CG_Reference.png)
+The coordinate system for motor positions is FRD (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
+The origin of the system is the vehicle centre-of-gravity (COG) (**not** the autopilot location).
 
-### Control Surface direction (Conceptual)
+![Actuators CG reference diagram](../../assets/config/actuators/quadcopter_actuators_cg_reference.png)
+
+#### Bidirectional Motors
+
+Some vehicles may use bidirectional motors (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half).
+For example, ground vehicles that want to move forwards and backwards, or VTOL vehicles that have pusher motors that go in either direction.
```

Comment:
What is this 'VTOL vehicle with pusher that can go in either direction' use case? Is this a real use case?

Reply:
Yes, according to Beat. I guess the pusher can act as an airbrake in that case.